Base/res/ladybird/inspector.css

```diff
@@ -174,6 +178,16 @@ body {
     overflow: auto scroll;
 }
 
+.tab-header {
+    position: sticky;
+    top: 0;
+    left: 0;
+    right: 0;
+    background-color: var(--tab-controls);
```

Comment:
(nit) Maybe just me, but I don't love the look of the select item having the same background color as the tab controls above it. Maybe just seems off to me since it's adjacent to it, so it looks like the box is growing/shrinking as we change tabs.

Reply:
Hmm. We do the same with the headings in the style-properties table. (Which now looks really odd because of the gap.) I'm giving it a line as a separator for now, maybe someone can find a better option later.